docs/content/_docs/query_language.html

```diff
@@ -244,52 +202,175 @@ <h4>Operators</h4>
+  </tr>
+
+  <tr>
+    <th scope="row">Tag True</th>
+    <td><pre><code class="language-hql">[true]</code></pre></td>
+    <td><pre><code class="language-json">[true]</code></pre></td>
+  </tr>
+
+  <tr>
+    <th scope="row">Tag False</th>
+    <td><pre><code class="language-hql">[false]</code></pre></td>
+    <td><pre><code class="language-json">[false]</code></pre></td>
+  </tr>
+  
+  <tr>
+    <th scope="row">RegEx On Tag</th>
+    <td><pre><code class="language-hql">&lt;a&gt; ~ &lt;b&gt;</code></pre></td>
+    <td><pre><code class="language-json">["~", &lt;a&gt;, &lt;b&gt;]</code></pre></td>
+  </tr>
+  
+  <tr>
+    <th scope="row">Raw</th>
+    <td>Custom filters</td>
+    <td><pre><code class="language-hql">q &lt;a&gt;</code></pre></td>
+    <td><pre><code class="language-json">["q", &lt;a&gt;]</code></pre></td>
+  </tr>
+  
+</table>
+</div>
+
+<h4 id="customfilters">
+  Custom Filters
+  <a class="link-to" href="#customfilters"><span class="glyphicon glyphicon-link"></span></a>
+</h4>
+
+<p>
+  When you need to select series using a more complex syntax than that available through the basic tag suggestions, you can create custom filters.
+</p>
+
+<div class="table-responsive">
+<table class="table table-bordered">
+  <tr>
+    <th></th>
+    <th>Description</th>
+    <th>Parsed</th>
+    <th>JSON</th>
+  </tr>
+
+  <tr>
+    <th scope="row">Simple String</th>
+    <td>Simple string without quotes.</td>
+    <td><pre><code class="language-hql">hello1234</code></pre></td>
+    <td><pre><code class="language-json">hello1234</code></pre></td>
+  </tr>
+
+  <tr>
+    <th scope="row">Simple String with Quotes</th>
+    <td>Simple string with quotes.</td>
+    <td><pre><code class="language-hql">"hello world $"</code></pre></td>
+    <td><pre><code class="language-json">"hello world $"</code></pre></td>
   </tr>
 
   <tr>
-    <th scope="row">Tag Does Not Start With</th>
-    <td><pre><code class="language-hql">&lt;a&gt; !^ &lt;b&gt;</code></pre></td>
-    <td><pre><code class="language-json">["not", ["^", &lt;a&gt;, &lt;b&gt;]]</code></pre></td>
+    <th scope="row">Tag Matches</th>
+    <td>Tag foo equals to bar.</td>
+    <td><pre><code class="language-hql">&lt;foo&gt; = &lt;bar&gt;</code></pre></td>
+    <td><pre><code class="language-json">["=", &lt;foo&gt;, &lt;bar&gt;]</code></pre></td>
+  </tr>
+
+  <tr>
+    <th scope="row">Tag Does Not Match</th>
+    <td>Tag foo does not equal to bar.</td>
+    <td><pre><code class="language-hql">&lt;foo&gt; != &lt;bar&gt;</code></pre></td>
+    <td><pre><code class="language-json">["not", ["=", &lt;foo&gt;, &lt;bar&gt;]]</code></pre></td>
   </tr>
 
   <tr>
     <th scope="row">Key Matches</th>
-    <td><pre><code class="language-hql">$key = &lt;a&gt;</code></pre></td>
-    <td><pre><code class="language-json">["key", &lt;a&gt;]</code></pre></td>
+    <td>Key equals bar.</td>
+    <td><pre><code class="language-hql">$key = &lt;bar&gt;</code></pre></td>
```

Comment:
The `$key` key is only special by our convention, but its basically a normal tag and there's been some effort to move away from treating it as a special object. I think we can get rid of the sections related to it and just have the standard tag ones - eg the `Tag Matches` two rows up does the same thing as `Key Matches` if you give it `$key` as the tag.